encoding/thrift/outbound.go

```diff
@@ -24,22 +24,25 @@ import (
 	"bytes"
 	"io/ioutil"
 
-	"github.com/yarpc/yarpc-go"
 	"github.com/yarpc/yarpc-go/transport"
 
 	"github.com/thriftrw/thriftrw-go/protocol"
 	"github.com/thriftrw/thriftrw-go/wire"
-	"golang.org/x/net/context"
 )
 
 // Client is a generic Thrift client. It speaks in raw Thrift payloads. The code
 // generator is responsible for putting a pretty interface in front of it.
 type Client interface {
-	Call(ctx context.Context, r *Request) (*Response, yarpc.Meta, error)
+	// Call the given Thrift method.
+	Call(method string, req *Request, body wire.Value) (wire.Value, *Response, error)
 }
 
 // Config contains the configuration for the Client.
 type Config struct {
+	// Name of the Thrift service. This is the name used in the Thrift file
+	// with the 'service' keyword.
+	Service string
```

Comment:
should we be explicit and call this `ThriftService` till we figure out common terminology?